### PR TITLE
[4.0] com_action_logs options

### PR DESCRIPTION
**Pull request by brianteeman**
Simple pr to change the IP logging option to use the switcher instead of the old style yes/no

### Before
image

### After
image

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on cbe44c3e7743fb7a51d29e66030037a86c6bb6c2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24671">issues.joomla.org/tracker/joomla-cms/24671</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cbe44c3e7743fb7a51d29e66030037a86c6bb6c2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24671">issues.joomla.org/tracker/joomla-cms/24671</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24671">issues.joomla.org/tracker/joomla-cms/24671</a>.</sub>

File: administrator/components/com_actionlogs/config.xml [hunk 1]
```diff
@@ -6,12 +6,11 @@
 			type="radio"
 			label="COM_ACTIONLOGS_IP_LOGGING_LABEL"
 			description="COM_ACTIONLOGS_IP_LOGGING_DESC"
-			class="btn-group btn-group-yesno"
+			class="switcher"
 			default="0"
-			filter="integer"
```
Comment (wilsonge) [inline administrator/components/com_actionlogs/config.xml]: We still should have the filter here

Comment (brianteeman) [reply administrator/components/com_actionlogs/config.xml]: Why? Please educate me

Comment (brianteeman) [reply administrator/components/com_actionlogs/config.xml]: I don't understand the need if there is only two options, but will happily put it back if required

Comment (wilsonge) [reply administrator/components/com_actionlogs/config.xml]: It’s required. Ideally pretty much all the xml fields should have basic validation. It helps prevent against anyone submitting dodgy data (as we move towards web services this becomes higher risk than when it was simply the admin interface)

Comment (brianteeman) [reply administrator/components/com_actionlogs/config.xml]: Ok thanks

Comment (mbabker) [reply administrator/components/com_actionlogs/config.xml]: The filters will also force data to be saved in the correct type, here you actually want to save an integer and not a numeric string (well actually you'd want it to be boolean true/false but somewhere between the Form API and MVC library things just blow up so integer's second best choice) so the integer filter will both make the data the appropriate type and offer some basic protection against someone trying something funny with either DOM manipulation or a web services API.

File: administrator/components/com_actionlogs/config.xml [hunk 2]
```diff
@@ -6,12 +6,12 @@
 			type="radio"
 			label="COM_ACTIONLOGS_IP_LOGGING_LABEL"
 			description="COM_ACTIONLOGS_IP_LOGGING_DESC"
-			class="btn-group btn-group-yesno"
+			class="switcher"
 			default="0"
 			filter="integer"
-			>
-			<option value="1">JYES</option>
+		       >
```
Comment (richard67) [inline administrator/components/com_actionlogs/config.xml]: Line 12 use tabs before `>`.

**Review by richard67 — CHANGES_REQUESTED** [review]

File: administrator/components/com_actionlogs/config.xml (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_actionlogs/config.xml]: thanks

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 02c78e228c12790790ac2db164cd061637cbfa0a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24671">issues.joomla.org/tracker/joomla-cms/24671</a>.</sub>

**Review by richard67 — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**